db/seeds/posts/articles.html

```diff
@@ -3,10 +3,10 @@
 <p>Currently, we have two main post types: Q&amp;A (questions and answers), and articles.</p>
 <p>Q&amp;A is what forms, in most cases, the majority of the content on the site. Both the default Q&amp;A and Meta categories use the Q&amp;A post type.</p>
 <p>One of the main technical differences between articles and Q&amp;A is that articles can&#39;t be answered, as well as having a <code>/articles/</code> path in the post URL. (Q&amp;A uses a <code>/questions/</code> path.)</p>
-<p>Articles can be used in several different ways. Some sites have a blog, where people can post about topics that are of interest to that community, as well as site updates (and, really, whatever that site decides). Other sites use articles to post information that doesn&#39;t quite fit in a Q&amp;A format - such as tutorials, reviews, or recommendations.</p>
+<p>Articles can be used in several different ways. Some communities have a blog, where people can post about topics that are of interest to that community, as well as site updates (and, really, whatever that site decides). Other communities use articles to post information that doesn&#39;t quite fit in a Q&amp;A format - such as tutorials, reviews, or recommendations.</p>
 <p><strong>Categories</strong></p>
-<p>By default, all sites start out with two main category types: Main Q&amp;A, which is where the main Q&amp;A content for the site goes; and Meta, where people can discuss the site itself. </p>
-<p>In additon, many sites have additional categories. For instance, <a href="https://writing.codidact.com">Writing</a> has a category for Writing Challenges, <a href="https://cooking.codidact.com">Cooking</a> has a category for sharing Recipes, and <a href="https://meta.codidact.com">Codidact Meta</a> has a category for a <a href="https://meta.codidact.com/categories/30">site Blog</a> (where the Codidact Team shares updates).</p>
+<p>By default, all communities start out with two main category types: Main Q&amp;A, which is where the main Q&amp;A content for the site goes; and Meta, where people can discuss the site itself. </p>
```

Comment:
Missing replacement in L1, 12

Reply:
I just did a find and replace from sites > communities for this file - let me know with any suggested specific changes if that's not what you want

Reply:
Did the "in addition" line go away?  The diff here shows it as deleted; it shouldn't be.  (And to match the other changes, it should be "many communities have additional..." rather than sites.)

Reply:
Nothing got deleted - the addition had a spelling error which I went and fixed - if you want feel free to check the files changed tab on this PR as it shows the net changes made in this PR